api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/advancedhivillness/ListOfPatientsOnAdvancedHivIllnessQueries.java

```diff
@@ -151,11 +151,43 @@ public String getCd4ResultBellowOneYear() {
    * @return {@link String}
    */
   public String getCd4ResultOverOrEqualTo5y() {
-    return getPatientsWithCD4AbsoluteResultOnPeriodQuery(200, false)
+    return " SELECT ps.person_id, o.value_numeric as cd4  "
+        + "FROM   person ps  "
+        + "           INNER JOIN encounter e  "
+        + "                      ON ps.person_id = e.patient_id  "
+        + "           INNER JOIN obs o  "
+        + "                      ON e.encounter_id = o.encounter_id  "
+        + "INNER JOIN ( "
+        + getLastCd4ResultDateQueries()
+        + " ) last_cd4 ON last_cd4.person_id = ps.person_id  "
+        + " WHERE  ps.voided = 0  "
+        + "  AND e.voided = 0   "
+        + "  AND o.voided = 0  "
+        + "  AND e.encounter_type IN ( ${6}, ${13}, ${51} )  "
+        + "  AND o.concept_id = ${1695}  "
+        + "  AND o.value_numeric < 200  "
```

Comment:
same as the other ticket, please use parameters and if possible use the same method